### ISSUE TITLE
Bug: VoiceOver does not read the current chapter title in the player screen

### ISSUE THREAD

**Issue by Simon818** [TortugaPower/BookPlayer#1048]
## Environment

 - OS Version: iOS (any, currently on 17.2)
 - App Version: 5.0.10-18
- This has been an issue as long as I've used the app, which is at least four months. My bad for not reporting, just wanted to be clear this is not new.

## Description

VoiceOver does not read the part of the player screen that shows the title of the currently-playing chapter. I have to go to the control center to find out which chapter I'm on.

## Steps to reproduce the behavior
1. Turn on VoiceOver while the full-screen player is open on a multi-file book, such as an m4b or a volume. You'll need one with distinct chapter titles to really illustrate this point.
2. Swipe up with two fingers to make VoiceOver read the entire screen top to bottom. VoiceOver will never speak the name of the current chapter, only the title of the book/volume and the chapter index and count.

For instance, in the book I'm reading, There are several tracks before the first chapter. If I'm on the epigraph, VoiceOver will tell me I'm on chapter 2 of 150, but will not read the "Epigraph" title. I can read it in control center, on the lock screen, and if I run a screenshot of the app through OCR, so I know it is being picked up by the app.

**Comment by Simon818** [comment]
Hey, just wondering if this was seen? It's really the only complaint I currently have, and there's lots of really nice VoiceOver improvements, so I thought I'd check. (Thanks for those, by the way.)

**Comment by GianniCarlo** [comment]
@Simon818 thanks for bumping this issue, this should be a quick fix, I'll look into it and see if I can include it in the next release

**Comment by Simon818** [comment]
Thanks, I try not to bump things too often but I am a person who constantly forgets about / misses things, "and I knew it probably wasn't a complex fix. I really like the idea of having that text control right in between the chapter navigation buttons in swipe order, so when swiping right with VoiceOver, it gets read out after the "previous chapter" button. But I'll leave that up to you.

**Comment by GianniCarlo** [comment]
no worries @Simon818 , I checked and actually had it disabled for VoiceOver, because the chapter title also works as a button to toggle between chapter context and book context, and at the time I must have thought that we already have that functionality for the progress button, but the chapter title button is inferior because it does not change upon activation. In any case, I enabled it, and removed the accessibility trait that marks it as a button, it is still actionable, but users won't be encouraged to activate it, and three elements down, the same functionality is available via the progress button

I'll include the fix in the next release 👌

**Comment by GianniCarlo** [comment]
This is now live on the AppStore
